hpx/runtime/agas/big_boot_barrier.hpp

```diff
@@ -67,7 +67,7 @@ struct HPX_EXPORT big_boot_barrier : boost::noncopyable
     };
 
     big_boot_barrier(
-        parcelset::parcelport& pp_
+        parcelset::parcelport* pp_
       , parcelset::endpoints_type const& endpoints_
```

Comment:
I now think this was part of my change to resolve the circular dependency between the initialization of AGAS and the parcel handler.